src/views/Dashboard/components/Widgets/BlocksWidget/BlocksWidget.tsx

```diff
@@ -147,8 +148,8 @@ const BlocksWidget: React.FC = () => {
       if (detailedBlock) {
         const txDetailsRows: Array<ITableRow> | undefined = detailedBlock?.transactions.map((tx) => {
           return {
+            date: new Date(tx.timestamp * 1000 + JUPGenesisTimestamp * 1000).toLocaleString(userLocale.localeStr, userLocale.options),
```

Comment:
fyi: https://github.com/Up-North-Coding/jovian/issues/98#issuecomment-1220177604

Reply:
yessir, handled in another branch